module.c

```diff
@@ -548,6 +548,9 @@ static JSValue js_c_function(
 	JSValue js_result = JS_NULL;
 	if (python_to_quickjs_possible(context, result)) {
 		js_result = python_to_quickjs(context, result);
+	} else {
+		PyErr_Clear();
+		js_result = JS_ThrowInternalError(ctx, "Can not convert Python result to JS.");
```

Comment:
Perhaps worth changing `python_to_quickjs_possible` to return an error message string instead of doing this "set Python exception, clear Python exception, set JS exception" dance, which also loses the accurate error message

Reply:
Agreed that could we a good refactor. Let's not do that in this PR though.